eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -283,14 +306,31 @@ def emceefitter(lc, model, meta, log, **kwargs):
     sampler.run_mcmc(pos, meta.run_nsteps, progress=True)
     samples = sampler.get_chain(flat=True, discard=meta.run_nburn)
 
-    medians = []
-    for i in range(len(step_size)):
-            q = np.percentile(samples[:, i], [16, 50, 84])
-            medians.append(q[1])
-    fit_params = np.array(medians)
-
+    # Record median + percentiles
+    q = np.percentile(samples, [16, 50, 84], axis=0)
+    fit_params = q[1] # median
+    
+    # Create table of results
+    t_results = table.Table([freenames, fit_params, q[0]-q[1],q[2]-q[1]], 
+                            names=("Parameter", "Median", "-1sig", "+1sig"))
```

Comment:
Should we relabel to 16th and 84th percentiles?

Reply:
I think I should add "16th perc" and "84th perc" because what I'm saving is 84th perc-50th perc and 16th perc-50th perc. Would that work for you?

Reply:
Sorry, was thinking mean but typed median.  To be clear... I was suggesting mean and +/-1 sigma about the mean